monitoring/mongodb/dashboard.py

```diff
@@ -0,0 +1,1397 @@
+from grafanalib.core import (
+    ConstantInput,
+    DataSourceInput,
+    GridPos,
+    Heatmap,
+    HeatmapColor,
+    Repeat,
+    RowPanel,
+    Template,
+    Templating,
+    Threshold,
+    YAxis,
+)
+from grafanalib import core
+from grafanalib import formatunits as UNITS
+from scalgrafanalib import (
+    layout,
+    metrics,
+    Dashboard,
+    GaugePanel,
+    PieChart,
+    Stat,
+    Target,
+    TimeSeries,
+    Tooltip,
+    StateTimeline,
+)
+
+# Constants
+DATASOURCE = "${DS_PROMETHEUS}"
+NAMESPACE = "zenko"
+
+# Job definitions
+JOBS = {
+    "configsvr": "zenko/data-db-mongodb-sharded-configsvr",
+    "mongos": "zenko/data-db-mongodb-sharded-mongos",
+    "shards": "zenko/data-db-mongodb-sharded-shards-data",
+    "shard_pattern": "zenko/data-db-mongodb-sharded-shard.*-data",
+    "all_shards": ".*shards.*",
```

Comment:
I don't understand the 3 kinds of job for shards : `job` is used to filter based on the prometheus job which performed the scrapping (which maps to the service/servicemonitor in k8s), so either we should have only one way to identify metrics from shard servers... unless this is used accross different metrics, and it should probably be in separate variables/arrays/... to make it clear what the values relate to.

Reply:
I see the code is outdated, but no change: some things were not pushed?